src/reflect/scala/reflect/runtime/Settings.scala

```diff
@@ -62,4 +62,7 @@ private[reflect] class Settings extends MutableSettings {
   val Yrecursion        = new IntSetting(0)
   def isScala212        = true
   private[scala] def isScala213 = true
+
+  @deprecated("Use Vprintpos instead", "2.13.1")
+  val Xprintpos: BooleanSetting = Vprintpos
```

Comment:
The awful reflect layer is an excellent reason not to make this otherwise-trivial change.